src/qb4-packw/kr-neoni8mm.c.in

```diff
@@ -0,0 +1,193 @@
+// Copyright 2023 Google LLC
+//
+// This source code is licensed under the BSD-style license found in the
+// LICENSE file in the root directory of this source tree.
+
+$ABC = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ"
```

Comment:
bit avoid $ABC moving forward.  replace
${ABC[N:N+4]}
with
${N//4}
this is because future cpus have very large vectors, we have 80 column code style, and sve & rvv dont have known vector sizes.
If you have complicated lanes dont worry about it, but it ${N} or ${N//4} works its an easy cleanup